astropy/cosmology/connect.py

```diff
@@ -1,9 +1,16 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
 from astropy.cosmology import units as cu
 from astropy.io import registry as io_registry
 from astropy.units import add_enabled_units
 
+if TYPE_CHECKING:
+    from astropy.cosmology import Cosmology
```

Comment:
This is reminding me that there are one or two places in PlasmaPy that I need to go back and add `if TYPE_CHECKING` blocks.  Thank you for requesting a review and giving me an opportunity to learn something!